Editor/Scripts/Startup/RegressionPackagePopup.cs

```diff
@@ -360,30 +360,33 @@ private bool IsURPEnabled()
         return false;
     }
     
-    private void ImportSample()
+    private void ImportSample(string sampleName)
     {
         string packageName = "gg.regression.unity.bots";
-        string samplePath = "Samples~/ThirdPersonDemoURP";
-        string destinationPath = "Assets/ThirdPersonDemoURP";
+        string samplePath = "Samples~/" + sampleName;
+        string destinationPath = "Assets/" + sampleName;
 
         // Construct the path to the sample within the package
-        string packagePath = Path.Combine("Packages", packageName, samplePath);
-
+        string packagePath = "Packages/" + packageName + '/' + samplePath;
```

Comment:
I think we should stick with using `Path.Combine` instead of assumed `/` pathing.

the reason it wasn't working before is that the first argument must be an absolute path.

something like this

```csharp
string samplePath = Path.Combine(FileUtil.GetPhysicalPath("Samples~"), "ThirdPersonDemoURP");
string destinationPath = Path.Combine(FileUtil.GetPhysicalPath("Assets"), "ThirdPersonDemoURP");
string packagePath = Path.Combine(FileUtil.GetPhysicalPath("Packages"), packageName, samplePath);

//...


EditorSceneManager.OpenScene(Path.Combine(destinationPath, "Demo", "Scenes", "Playground.unity"));
```